main/webapp/modules/core/MOD-INF/controller.js

```diff
@@ -250,6 +251,8 @@ function registerImporting() {
   IM.registerExtension(".ods", "text/xml/ods");
   
   IM.registerExtension(".nt", "text/rdf/nt");
+  IM.registerExtension(".ntriples", "text/rdf/nt");
```

Comment:
@wetneb this is because of stupid DBpedia.  
![image](https://user-images.githubusercontent.com/986438/79052140-0ee2b380-7bfa-11ea-95ec-7d019bd64920.png)